README.md

```diff
@@ -279,16 +282,17 @@ The comprehensive reading list for Linux kernel-space programming. The choice is
 * [Linux Kernel Networking: Implementation and Theory ](https://www.amazon.com/dp/143026196X) (Rami Rosen, 2013)
 * [Linux Kernel Programming](https://www.amazon.com/dp/178995343X) (Kaiwan N Billimoria, 2021)
 * [Linux Kernel Programming Part 2 - Char Device Drivers and Kernel Synchronization](https://www.amazon.com/dp/180107951X) (Kaiwan N Billimoria, 2021)
+* [Linux Kernel Debugging](https://www.amazon.com/dp/1801075034) (Kaiwan N Billimoria, 2022)
 * [Mastering Linux Device Driver Development](https://www.amazon.com/dp/178934204X) (John Madieu, 2021)
-* [Linux Device Drivers Development: Develop customized drivers for embedded Linux](https://www.amazon.com/dp/1785280007) (John Madieu, 2017)
```

Comment:
The great device driver book authored by John Madieu, I can't express its importance, hopefully now it's updated to its next edition.